pulsar/activity/metadata.go

```diff
@@ -0,0 +1,25 @@
+package activity
+
+import "github.com/project-flogo/core/data/coerce"
+
+type Settings struct {
+	Connection interface{} `md:"connection"`
```

Comment:
Should be `connection.Manager` @fm-tibco

Reply:
The reason I added just `interface{}` is because `connection` in the json can either be:

```
"connection" : "conn://kafka"
```
or
```
"connection": {
  "ref":"github.com/..../kafka/connection",
  "settings":{
   "brokerUrl" : "localhost:9092"
   }
}
```

Reply:
It should be connection.Manager, the engine should resolve that properly.